tests/functional/sponsorship/sponsorship_tests.py

```diff
@@ -107,29 +104,7 @@ def test_ott(self, extra_assert_tests, ott, extra_assert_tests_from_another_brow
         assert has_linkouts(browser, include_site_internal=True) == False
         assert self.zoom_disabled()
         print(" ", end="", flush=True)
-
```

Comment:
not sure what this unused function was about. Looks like just an unfinished idea for a test?

Reply:
Hmm, looks like you are right, or at least that there's a stray print there which we should remove.